docs/source/how-to/migrations/upgrade-to-16.md

```diff
@@ -0,0 +1,70 @@
+# Upgrading your OAuthenticator to version 16.0
+
+The following section describes what to keep in mind when upgrading to OAuthenticator 16.0.
+
+````{important}
+(token_request)=
```

Comment:
Is this important section providing a background to understand other things in the section below?

It seems like descriptions on how things work that may not belong in a version upgrade guide. The technical level here is very high, so I worry a bit that a very limited audience can make use of it.

Reply:
> Is this important section providing a background to understand other things in the section below?

It's here more in order to don't repeat this explanation and the links to outside resources multiple times in the sections below.

And also because when reading through the RFC standards, I noticed that for example what we call `token_request` it's called `access token request` there and wanted to make sure people know they're the same thing and I'm not confusing them with possible new names?

Reply:
Maybe you could add some introduction sentence between "Important" and the first rubric?

We want this to be something people dare keep reading about, and this is "upgrade instructions", so end users shouldn't need to know the details _unless we have a breaking change we must explain to them_. Currently, its like a snippet of facts and I don't know if its relevant if I'd upgrade and I used for example the GitHubOAuthenticator, etc.

So I think we need some kind of guidance for this notice if its to remain at this location, to clarify for whom this is relevant to read and try to understand.

Reply:
Good point @consideRatio. I've just added a commit that adds these two "definitions" into  footnotes to be less intrusive, but also at hand in case someone wants to double check their meaning:

Check it out rendered at https://oauthenticator--569.org.readthedocs.build/en/569/how-to/migrations/upgrade-to-16.html